test/api/peers.js

```diff
@@ -483,7 +478,7 @@ describe('GET /api/peers', function () {
 	});
 });
 
-describe('GET /api/peers/get', function () {
+describe.skip('GET /api/peers/get', function () {
```

Comment:
Why are we skipping peers/get tests now?

Reply:
quoting commit message: `skip part of peers tests relying on existing peer (fixed in #710)`